docs/gambit/mutation-verifier.md

```diff
@@ -36,12 +36,21 @@ Once you have updated your `certora-cli` installation using `pip` to get the rel
 dependencies, run Gambit from the command line:
 
 ```
-certoraMutate --prover_conf path/to/prover/prover.conf --gambit_conf path/to/gambit/gambit.conf
+certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/mutation/mutation.conf
+```
+
+```{note}
+You must run `certoraMutate` from the root of the Solidity project directory.
+The files `prover.conf` and `mutation.conf`
+can be in their own directories.
```

Comment:
There is an unnecessary line break before this

Reply:
not in the docs though?